portfolio/src/main/webapp/script.js

```diff
@@ -76,16 +78,73 @@ function createCommentElement(fname, surname, date, message, blobKey) {
   if (!(blobKey == null || blobKey == "")) {
     // Fetch blob from GetBlobServlet.
     blobUrlString = "/get-blob?blob-key=" + blobKey;
-    console.log(blobKey);
     fetch(blobUrlString)
         .then((image) => {
-          // Create an img element in the HTML document using the fetched image's URL.
+          // Create a slider box element in the HTML doc using the fetched img's URL.
+          const slider = document.createElement("div");
+          slider.className = "slider";
+          commentElement.appendChild(slider);
+
+          // Create img elements for each slide in the HTML document using the fetched image's URL.
           const img = document.createElement("img");
           img.src = image.url;
-          console.log(image.url);
           img.alt = "Comment picture."
-          img.style = "max-width: 40%;";
-          commentElement.appendChild(img);
+          const img2 = document.createElement("img");
+          img2.src = image.url;
+          img2.alt = "Comment picture."
+
+          // Create slide showing img and a table displaying image labels and confidence scores.
+          sliderBox1 = document.createElement("div");  // For image labels
+          sliderBox1.className = "slide";
+          sliderBox1.appendChild(img);
+          table = document.createElement("table");
+          
+          // Building header row.
+          tableHeader = document.createElement("thead");
+          headerLabels = document.createElement("tr");
+          headerLabels.innerHTML = "<th>Image Label</th><th>Confidence</th>"
+          tableHeader.appendChild(headerLabels);
+          table.appendChild(tableHeader);
+ 
+          // Building body rows. 
+          tableBody = document.createElement("tbody");
+          for (i = 0; i < imageDescs.length; i++) {
+            row = document.createElement("tr");
+            row.innerHTML = "<td>" + imageDescs[i] + "</td><td>" + descScores[i] + "</td>";
```

Comment:
Should format this number a bit better - having 10+ decimals isn't really relevant to most visitors.